tendrl/gluster_integration/flows/update_brick_status/__init__.py

```diff
@@ -0,0 +1,25 @@
+import etcd
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+
+
+class UpdateBrickStatus(flows.BaseFlow):
+    def run(self):
+        logger.log(
+            "info",
+            NS.publisher_id,
+            {"message": "starting flow for updating brick status"}
+        )
+        super(UpdateBrickStatus, self).run()
+        node = self.parameters.get("Node.fqdn")
+        cluster_id = self.parameters.get("TendrlContext.integration_id")
+        self.update_brick_status(node, cluster_id)
+
+    def update_brick_status(self, node, cluster_id):
+        bricks = NS.gluster.objects.Brick(fqdn=node).load_all()
+        for brick in bricks:
+            brick.status = "Stopped"
```

Comment:
If status is `Stopped` hard coded why call the flow as `UpdateBrickStatus`?
Or else pass status as `Stopped` as parameter to the flow from the caller.

Reply:
ack

Reply:
Yes, this flow needs to either "stop_brick" 

or

 "update_brick_status" and handle all statuses